gnovm/cmd/gno/run.go

```diff
@@ -76,6 +78,13 @@ func (c *runCmd) RegisterFlags(fs *flag.FlagSet) {
 		"",
 		"enable interactive debugger using tcp address in the form [host]:port",
 	)
+
+	fs.StringVar(
+		&c.pkgPath,
+		"pkgpath",
+		"",
+		"value of realm pkgPath, for testing purpose",
```

Comment:
What if we automatically guess it from the gno.mod file? Alternatively, we could keep this flag but not label it as "for testing." Instead, it could serve as a way to simulate running the same code from a different location, which could make sense, like a "package fork."

Reply:
Yes, if a gno.mod file exists, the `gno run -debug` should be able to derive the default package path to apply. The `-pkgpath` flag is still useful for the simulation use case.